man/systemd.naming-scheme.xml

```diff
@@ -0,0 +1,403 @@
+          <literal>v<replaceable>port</replaceable></literal>, where <replaceable>port</replaceable> is the
+          virtual device number. Previously those virtual devices were named as if completely independent.
+          </para>
+
+          <para>The ninth and later NPAR virtual devices are named following the scheme used for the first
+          eight NPAR partitions. Previously those devices were not renamed and the kernel default
+          ("eth<replaceable>N</replaceable>") was used.</para>
+
+          <para>Names are also generated for PCI devices where the PCI network controller device does not
+          have an associated slot number itself, but one of its parents does. Previously those devices were
+          not renamed and the kernel default was used.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><constant>v240</constant></term>
+
+          <para>The <literal>ib</literal> prefix and stable names for infiniband devices are
+          introduced. Previously those devices were not renamed.</para>
+
+          <para>The ACPI index field (used in <varname>ID_NET_NAME_ONBOARD=</varname>) is now also used when
+          0.</para>
+
+          <para>A new naming policy <varname>NamePolicy=keep</varname> was introduced. With this policy, if
+          the network device name was already set by userspace, the device will not be renamed
+          again. Previously, this naming policy applied implicitly, and now it must be explicitly
+          requested. Effectively, this means that network devices will be renamed according to the
+          configuration, even if they have been renamed already, if <constant>keep</constant> is not
+          specified as the naming policy in the <filename noindex='true'>.link</filename> file. See
+          <citerefentry><refentrytitle>systemd.link</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+          for a description of <varname>NamePolicy=</varname>.</para>
+        </varlistentry>
+          
+        <varlistentry>
+          <term><constant>v243</constant></term>
+
+          <para>Support for netdevsim (simulated networking devices) was added. Previously those devices were
+          not renamed.</para>
+        </varlistentry>
+
+        <para>Note that <constant>latest</constant> may be used to denote the latest scheme known (to this
+        particular version of systemd.</para>
+    </variablelist>
+  </refsect1>
+
+  <refsect1>
+    <title>Examples</title>
+
+    <example>
+      <title>PCI Ethernet card with firmware index "1"</title>
+
+      <programlisting>ID_NET_NAME_ONBOARD=eno1
+ID_NET_NAME_ONBOARD_LABEL=enEthernet Port 1
+      </programlisting>
+      <!-- FIXME: nuke the prefix! -->
+    </example>
+
+    <example>
+      <title>PCI Ethernet card in hotplug slot with firmware index number</title>
+
+      <programlisting># /sys/devices/pci0000:00/0000:00:1c.3/0000:05:00.0/net/ens1
```

Comment:
Maybe show is as command line:

```
devadm test-builtin net_id /sys/class/net/enp0s31f6
```

Though I'd suggest to suppress the noise that outputs before and after the properties